### ISSUE TITLE
Server crashes on launch

### ISSUE THREAD

**Issue by phajtol** [lenosisnickerboa/csgosl#217]
Hello,

I was using csgosl regularly a few months ago and yesterday I launched it again after a longer time. It asked me to update to 2.4.7 and then I also updated server to latest version. After that, server no longer launches. It shows console, but gets stuck on SteamDatagramServer_Init succeeded (see screen) and then quits. I have tried installing version 2.7 that I was running before but same issue happens. I am running Windows 10 Enterprise version 1809 build 17763.775. My PC has Intel Core i5-9600K on Gigabyte Z390 Gaming X with MSI GeForce RTX 2070 Armor 8G and Kingston HyperX 16 GB RAM (2x8) DDR4 2666 MHz CL16. I can provide addition info if neccessary.

Thank you for any help.

![Desktop Screenshot 2019 10 04 - 22 35 31 84 (2)](https://user-images.githubusercontent.com/27435127/66244491-f5235980-e708-11e9-8d9e-1a2878b6a14f.png)

**Comment by lenosisnickerboa** [comment]
AFK, but some first questions; are you running any extra stuff you added
yourself? Do yo have sourcemod enabled? If yes, what plugins?

One thing to consider is to reinstall a new server and see if its any of
the settings causing the issues. A bit brute force, I know...

**Comment by phajtol** [comment]
So it seems that your suggested solution to install new instance of server has helped, new server works good.

Now the question is what is causing the problem. My idea is that it may have something to do with Windows Firewall but I am not sure and I will try to find it. If I find something noteworthy, I will let you know. 

Thanks for your help!

**Comment by lenosisnickerboa** [comment]
Ok, thanks! Please reopen a new issue or append your info to this issue if you find anything.

**Comment by phajtol** [comment]
Hi there!

So after some digging I found exactly what was causing the crash. It is file `cfg/steam.cfg`, and command `gameserverlogintoken` in it. My token has expired, and that's why it was crashing. If I set empty string as a parameter, it goes fine but is obviously limited to LAN only. If I put random string into it, it said something like cannot connect to Steam and it stopped there (console window stays open with the error). But if I put expired one into it, it crashed like I described above.

If you are interested in any further info or something contact me.

Thanks for making such good software!
